### PR TITLE
Calculate future restart time dynamically

### PR DESCRIPTION
**Pull request by CyberDem0n**
`do_POST_restart` was ramdomly showing not 100% coverage after 2016-08-20 due to hardcoded timestamps.

### CODE REVIEW

**Comment by feikesteenbergen** [comment]
👍

**Comment by alexeyklyukin** [comment]
👍

File: tests/test_api.py [hunk 1]
```diff
@@ -198,26 +199,26 @@ def make_request(request):
         request = make_request('foobar=baz')
         MockRestApiServer(RestApiHandler, request)
         # wrong role
-        request = make_request('{"schedule": "2016-08-20 12:45TZ+1", "role": "unknown", "postgres_version": "9.5.3"}')
+        request = make_request(schedule=future_restart_time.isoformat(), role='unknown', postgres_version='9.5.3')
```
Comment (alexeyklyukin) [inline tests/test_api.py]: Where is the future_restart_time defined?

**Comment by alexeyklyukin** [comment]
👍

**Pull request merged**